README.md

```diff
@@ -29,18 +37,25 @@ Or go to _Settings → Install_ and search for `prettier-atom`.
 
 Make sure to restart Atom after the package is installed.
 
-## Usage
+## Detailed Usage
 
 There are two modes you can use:
 
 * Invoke manually using the **keyboard shortcut** (if no selection, whole file is formatted): <kbd>CTRL</kbd> + <kbd>ALT</kbd> + <kbd>F</kbd>
 * Automatically **format on save** (requires enabling in _Packages → Prettier → Toggle Format on Save_)
 
+Note this plugin will default to using the prettier in your project's 'node_modules' directory.  If none is found then
+it falls back to using whatever version is installed with the plugin.
+
```

Comment:
Can we try this text?:

`Note: by default, we use the prettier instance in your project's 'node_modules' directory.  If one isn't found, then we fall back to using the version that comes bundled with the prettier-atom package.`